src/main/java/edu/pio/chinczyk/GameController.java

```diff
@@ -105,6 +105,19 @@ private void bindPawns() {
         pawns[3][3] = red_pawn_4;
     }
 
+    private void bindPawnsToModel() {
+        for(int playerID = 0; playerID < MAX_PLAYERS; ++playerID) {
+            for(int pawnID = 0; pawnID < PAWNS_PER_PLAYER; ++pawnID) {
+                Pawn pawn = ((LudoApp)this.getApp()).getBoard().getPlayer(playerID).getPawn(pawnID);
```

Comment:
((LudoApp)this.getApp()).getBoard() na zewnatrz petli
.getPlayer() do pierwszej petli